### DISCUSSION TITLE
음식점 데이터 테이블 분리 제안

### DISCUSSION THREAD

**Discussion by TaeyeonRoyce** [woowacourse-teams/2023-celuveat#83] [Ideas]
## 현재
Admin에서 영상 등록을 위해 조회하는 음식점 테이블과 실제 서비스에 활용되는 음식점 테이블을 같이 사용하고 있음

## 개선 사항
Admin에서 검색용 DB와 서비스에서 사용 되는 음식점 DB를 분리해서 저장하면 어떨까요
대략 120,000개의 row인데 다 사용하지도 않아서 분리하면 많이많이 좋을 것 같습니다

**Comment by shin-mallang** [comment]
오 이거 좋은데, 레벨4때 성능개선으로 남겨두는거 어떤가요?

**Comment by kdkdhoho** [comment]
저는 지금 해도 좋고 레벨4때 해도 좋습니다~

**Comment by odo27** [comment]
저도 지금 해도 좋고 레벨4에 해도 좋습니다!
